front/src/core-logic/domain/admin/icUsersAdmin/icUsersAdmin.slice.ts

```diff
@@ -23,7 +24,8 @@ export type NormalizedIcUserById = Record<
 
 type IcUsersAdminFeedbackKind =
   | "usersToReviewFetchSuccess"
-  | "agencyRegisterToUserSuccess";
+  | "agencyRegisterToUserSuccess"
+  | "agencyRejectToUserSuccess";
```

Comment:
le nomage on l'avait revu je crois. Eux sont passé entre les mailles. Qu'est-ce que tu pense de :
```suggestion
  | "agencyRejectionForUserSuccess"
```